move_group: manipulator  # name of the move group

display_path: #number of the path to be displayed
- path1
- path2

path1: # example of path
  - [ 0, 6.81795190256034e-06, -2.2491692595197703, 1.7835064169957895, 0.35060406706291775, 7.994698305739827e-06, -1.4557505040681777 ]
  - [ 0.1, 6.81795190256034e-06, -2.2491692595197703, 1.7835064169957895, 0.35060406706291775, 7.994698305739827e-06, -1.4557505040681777 ]
  - [ 0.2, 6.81795190256034e-06, -2.2491692595197703, 1.7835064169957895, 0.35060406706291775, 7.994698305739827e-06, -1.4557505040681777 ]
  - [ 0.3, 6.81795190256034e-06, -2.2491692595197703, 1.7835064169957895, 0.35060406706291775, 7.994698305739827e-06, -1.4557505040681777 ]
  - [ 0.4, 6.81795190256034e-06, -2.2491692595197703, 1.7835064169957895, 0.35060406706291775, 7.994698305739827e-06, -1.4557505040681777 ]
  - [ 0.5, 6.81795190256034e-06, -2.2491692595197703, 1.7835064169957895, 0.35060406706291775, 7.994698305739827e-06, -1.4557505040681777 ]
  - [ 0.6, 6.81795190256034e-06, -2.2491692595197703, 1.7835064169957895, 0.35060406706291775, 7.994698305739827e-06, -1.4557505040681777 ]
  - [ 0.7, 6.81795190256034e-06, -2.2491692595197703, 1.7835064169957895, 0.35060406706291775, 7.994698305739827e-06, -1.4557505040681777 ]
  - [ 0.8, 6.81795190256034e-06, -2.2491692595197703, 1.7835064169957895, 0.35060406706291775, 7.994698305739827e-06, -1.4557505040681777 ]
  - [ 0.9, 6.81795190256034e-06, -2.2491692595197703, 1.7835064169957895, 0.35060406706291775, 7.994698305739827e-06, -1.4557505040681777 ]
  - [ 1, 6.81795190256034e-06, -2.2491692595197703, 1.7835064169957895, 0.35060406706291775, 7.994698305739827e-06, -1.4557505040681777]

path2:
  - [0, 6.81795190256034e-06, -2.2491692595197703, 1.7835064169957895, 0.35060406706291775, 7.994698305739827e-06, -1.4557505040681777]
  - [0.5, 6.81795190256034e-06, 0, 0, 0.35060406706291775, 7.994698305739827e-06, -1.4557505040681777]
  - [1, 6.81795190256034e-06, -2.2491692595197703, 1.7835064169957895, 0.35060406706291775, 7.994698305739827e-06, -1.4557505040681777]
